pulseapi/profiles/test_views.py

```diff
@@ -521,3 +523,22 @@ def test_profile_view_is_active_filter(self):
         num_of_inactive_profiles = sum(profile['is_active'] is False for profile in entriesjson)
 
         self.assertEqual(num_of_inactive_profiles, 0)
```

Comment:
**Suggestion** We should add an assertion to check that the active profile is returned here (or that the number of active profiles is 1). That is the data that we are requesting.